src/audio/dai.c

```diff
@@ -542,7 +549,9 @@ static int dai_params(struct comp_dev *dev,
 	comp_dbg(dev, "dai_params()");
 
 	/* configure dai_data first */
-	dai_data_config(dev);
+	err = dai_data_config(dev);
+	if (!err)
```

Comment:
if (err < 0 ) ?   I validated it on my tgl.

Reply:
@RanderWang thanks, that's a time saver for me.